eclair-core/src/main/scala/fr/acinq/eclair/balance/Monitoring.scala

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2020 ACINQ SAS
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package fr.acinq.eclair.balance
+
+import kamon.Kamon
+import kamon.metric.Metric
+
+object Monitoring {
+
+  object Metrics {
+    val GlobalBalance: Metric.Gauge = Kamon.gauge("globalbalance", "Global Balance (BTC)")
+    val GlobalBalanceNormalized: Metric.Gauge = Kamon.gauge("globalbalance.normalized", "Global Balance Normalized")
+    val GlobalBalanceDiff: Metric.Gauge = Kamon.gauge("globalbalance.diff", "Global Balance Diff (Satoshi)")
+    val GlobalBalanceDetailed: Metric.Gauge = Kamon.gauge("globalbalance.detailed", "Global Balance Detailed (BTC)")
+  }
+
+  object Tags {
+    val BalanceType = "type"
+    val OffchainState = "state"
+    val DiffSign = "sign"
+
+    /** TODO: there is duplication with [[fr.acinq.eclair.blockchain.Monitoring]] */
```

Comment:
I think it's worth removing the `blockchain.Monitoring` one if we move the utxo check inside the `balance` package (as proposed in the comment inside `ZmqWatcher`).

Reply:
Done in 2b60dc9. I have kept the previous metrics because they offer a deeper level of details.